src/app/pages/system/general-settings/gui-form/gui-form.component.ts

```diff
@@ -86,6 +88,9 @@ export class GuiFormComponent {
     });
   }
 
+  format = IxFormatter.memorySizeFormatting;
+  parse = IxFormatter.memorySizeParsing;
```

Comment:
I think it's better not to do static functions. They definitely look neater, but they would be much harder to deal with in tests if we ever need to mock this service.